### ISSUE TITLE
Recruit Officers functions

### ISSUE THREAD

**Issue by falisca** [lanedirt/OGameX#229]
- Recruit officer menu

- Commanding staff

**Comment by Geda173** [comment]
Hi @lanedirt,

here, too, I am asking for a bit of guidance, as some of the officer functionalities are already baked into the game.
The commande for example features a couple functions that already exist, namely a) the fleet scanner (showing resources of incoming, foreign fleets), as well as the building queue. 

As far as I understand it, these features would have to be taken away at some point, as they are tied to the officers. Additionally, as this project is non-commercial, functions like the removal of ads don't have any real function. Should the commander still show the removal of ads in its tooltip/description as a sort of placeholder, or would you be okay with removing references that hold no real weight?

**Comment by lanedirt** [comment]
> As far as I understand it, these features would have to be taken away at some point, as they are tied to the officers.

Yes I agree. Some existing features like the build queue should ideally be paywalled behind the officers, and only be enabled when that officer is active. So just like the official game does it.

> Should the commander still show the removal of ads in its tooltip/description as a sort of placeholder, or would you be okay with removing references that hold no real weight?

I prefer to keep all game texts and explanations original, even if some things, like removal of ads, wouldn't really apply. We can always decide in the future to clean some things up if they really don't make sense, but for now I'd like to keep everything including the game texts 1-to-1 as much as possible.
